docs/modules/develop/pages/classes/commands.adoc

```diff
@@ -2,11 +2,214 @@
 
 A command is custom class that manages the logic between your input, usually a xref:develop:classes/forms.adoc[form] object, an object and an user.
 
-Usually, a command is used to create, update or delete a object.
+Usually, a command is used to handle the interaction between the controller and the model.
 
 Commands are located in the `app/commands/decidim/<my_module>` directory, and named: `<action>_<my_resource>.rb`.
 
-In the below example, you will be able to see how the command (`CreateMyResource`), events (`Decidim::MyModule::MyResourceEvent`) and jobs (`Decidim::MyModule::MyCustomJob`) can be used.
+Because most of the time, a command is used to create, update a resource, we are shipping 3 commands by default, which you can inherit by default in your own commands:
```

Comment:
```suggestion
Because most of the time, a command is used to create, update a resource, we are shipping 3 commands by default, which you can inherit  in your own commands:
```